plugin/src/main/java/com/denizenscript/denizen/tags/core/ServerTagBase.java

```diff
@@ -2698,14 +2019,416 @@ public static void adjustServer(Mechanism mechanism) {
+        // -->
+
+        // <--[tag]
+        // @attribute <server.delta_time_since_start>
+        // @returns DurationTag
+        // @deprecated use util.delta_time_since_start
+        // @description
+        // Deprecated in favor of <@link tag util.delta_time_since_start>
+        // -->
+
+        // <--[tag]
+        // @attribute <server.real_time_since_start>
+        // @returns DurationTag
+        // @deprecated use util.real_time_since_start
+        // @description
+        // Deprecated in favor of <@link tag util.real_time_since_start>
+        // -->
+
+        // <--[tag]
+        // @attribute <server.current_time_millis>
+        // @returns ElementTag(Number)
+        // @deprecated use util.current_time_millis
+        // @description
+        // Deprecated in favor of <@link tag util.current_time_millis>
+        // -->
+
+        // <--[tag]
+        // @attribute <server.has_file[<name>]>
+        // @returns ElementTag(Boolean)
+        // @deprecated use util.has_file
+        // @description
+        // Deprecated in favor of <@link tag util.has_file>
+        // -->
+
+        // <--[tag]
+        // @attribute <server.list_files[<path>]>
+        // @returns ListTag
+        // @deprecated use util.list_files
+        // @description
+        // Deprecated in favor of <@link tag util.list_files>
+        // -->
+
+        // <--[tag]
+        // @attribute <server.java_version>
+        // @returns ElementTag
+        // @deprecated use util.java_version
+        // @description
+        // Deprecated in favor of <@link tag util.java_version>
+        // -->
+
+        // <--[tag]
+        // @attribute <server.sql_connections>
+        // @returns ListTag
+        // @deprecated use util.sql_connections
+        // @description
+        // Deprecated in favor of <@link tag util.sql_connections>
+        // -->
+
+        // <--[tag]
+        // @attribute <server.scripts>
+        // @returns ListTag(ScriptTag)
+        // @deprecated use util.scripts
+        // @description
+        // Deprecated in favor of <@link tag util.scripts>
+        // -->
+
+        // <--[tag]
+        // @attribute <server.last_reload>
+        // @returns TimeTag
+        // @deprecated use util.last_reload
+        // @description
+        // Deprecated in favor of <@link tag util.last_reload>
+        // -->
+
+        // <--[tag]
+        // @attribute <server.stack_trace>
+        // @returns ElementTag
+        // @deprecated use util.stack_trace
+        // @description
+        // Deprecated in favor of <@link tag util.stack_trace>
+        // -->
+
+        // <--[tag]
+        // @attribute <server.debug_enabled>
+        // @returns ElementTag(Boolean)
+        // @deprecated use util.debug_enabled
+        // @description
+        // Deprecated in favor of <@link tag util.debug_enabled>
+        // -->
+        if (deprecatedServerUtilTags.contains(attribute.getAttributeWithoutParam(1))) {

```

Comment:
Rather than this, you should loop over the list and register an entry for each name (same for mechs)